src/users/users.controller.ts

```diff
@@ -0,0 +1,35 @@
+import {Body, Controller, Delete, Get, HttpCode, Param, Patch, Post} from '@nestjs/common';
+import {UsersService} from "./users.service";
```

Comment:
hmm we should fix or extend linter a bit, to have consistent import definitions (empty spaces between brackets).
will check it in different PR.

Reply:
Ok i will check linter in future PR.